### DISCUSSION TITLE
Hello everyone

### DISCUSSION THREAD

**Discussion by bipinKrishnan** [OpenMined/courses#73] [Introduce Yourself]
My name is Bipin and I am a recent mechanical engineering graduate. I've always wanted to join the openmined community as in my experience this is one of the most welcoming opensource communities out there. May be this course will help me join one of those awesome teams in the community. Thanks alot for all these great courses that you have made free for everyone.

**Comment by souravcipher** [comment]
Hi @bipinKrishnan! Welcome :wave: 

to join the OpenMined Teams check out the [OpenMined Placements](https://placements.openmined.org/) :rocket:

**Comment by Spartan-119** [comment]
Hello everyone!
I am Abin Varghese. I work as an ETA member in Data Science and Machine Learning track @ Infosys. I train clients and trainees on data science and machine learning.
I am excited to be here!

**Reply by yemikifouly** [reply]
Hello Abin! 🙂 Welcome to the discussion board! We are excited to have you here! 🙂 I hope you enjoy the course.
